web/src/beta/features/Editor/DataSourceManager/Asset/index.tsx

```diff
@@ -78,22 +85,12 @@ const Asset: React.FC<DataProps> = ({ sceneId, onSubmit, onClose }) => {
           label={t("Source Type")}
           description={t("Select the type of data source you want to add.")}>
           <SourceTypeWrapper>
-            <RadioButtonLabel>
-              <RadioButton
-                value="url"
-                checked={sourceType == "url"}
-                handleChange={c => c && setSourceType("url")}
-              />
-              {t("From URL")}
-            </RadioButtonLabel>
-            <RadioButtonLabel>
-              <RadioButton
-                value="value"
-                checked={sourceType == "value"}
-                handleChange={c => c && setSourceType("value")}
-              />
-              {t("From Value")}
-            </RadioButtonLabel>
+            <RadioGroup
+              options={DataSourceOptions}
+              layout="horizontal"
```

Comment:
If you do the above recommendation, then remove layout="horizontal" here

Reply:
done